### DISCUSSION TITLE
PyTorch 2.0 support?

### DISCUSSION THREAD

**Discussion by trimcrae** [deepspeedai/DeepSpeed#2765] [General]
Are there any plans to modify DeepSpeed to support PyTorch 2.0 once it has a stable release?

**Comment by tjruwase** [comment]
Yes, this is our plan. Thanks!

**Comment by bri25yu** [comment]
Is there a timeline for when PyTorch 2.0 will be supported? I'm excited to see potential training time improvements!

**Reply by SoonNOON** [reply]
Even with the copilot, you guys are slow on close bugs and pull requests; this is unacceptable.
